dns/jsonify/main.go

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/golang/glog"
+	"gopkg.in/yaml.v2"
+	"k8s.io/kubernetes/test/e2e/perftype"
+)
+
+const (
+	// S_TO_MS is a second to milisecond ratio.
+	S_TO_MS = 1000
+)
+
+// BenchamrkResult is a dns benchmark results structure.
+type BenchamrkResult struct {
+	Code   int             `yaml:"code"`
+	Data   BenchmarkData   `yaml:"data"`
+	Params BenchmarkParams `yaml:"params"`
+}
+
+// BenchmarkData represents dns benchmark data.
+type BenchmarkData struct {
+	Latency50Percentile float64 `yaml:"latency_50_percentile"`
+	Latency95Percentile float64 `yaml:"latency_95_percentile"`
+	Latency99Percentile float64 `yaml:"latency_99_percentile"`
+	AvgLatency          float64 `yaml:"avg_latency"`
+	MaxLatency          float64 `yaml:"max_latency"`
+	MinLatency          float64 `yaml:"min_latency"`
+	Qps                 float64 `yaml:"qps"`
+	QueriesCompleted    float64 `yaml:"queries_completed"`
+	QueriesLost         float64 `yaml:"queries_lost"`
+	QueriesSent         float64 `yaml:"queries_sent"`
+}
+
+// BenchmarkParams represents dns benchmark params.
+type BenchmarkParams struct {
+	RunLengthSeconds float64  `yaml:"run_length_seconds"`
+	QueryFile        string   `yaml:"query_file"`
+	KubednsCpu       *float64 `yaml:"kubedns_cpu"`
+	DnsmasqCpu       *float64 `yaml:"dnsmasq_cpu"`
+	DnsmasqCache     *float64 `yaml:"dnsmasq_cache"`
+	MaxQps           *float64 `yaml:"max_qps"`
+}
+
+func main() {
+	defer glog.Flush()
+	err := run()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func run() error {
+	var benchmarkDirPath string
```

Comment:
You could put these on the same line:
var benchmarkDirPath, jsonDirPath, benchmarkName string

Reply:
Done.